qpmodel/PhysicalNode.cs

```diff
@@ -1099,7 +1129,7 @@ public override string Exec(Func<Row, string> callback)
                     {
                         // output current grouped row if any
                         if (curGroupKey != null)
-                            FinalizeAGroupRow(context, keys, curGroupRow, callback);
+                            FinalizeAGroupRow(context, curGroupKey, curGroupRow, callback);
```

Comment:
I thought this is fixed a while back?

Reply:
yes I used the same branch, I don't know why this change still appears.